dbt/include/athena/macros/materializations/models/incremental/incremental.sql

```diff
@@ -22,7 +23,7 @@
   {% set old_tmp_relation = adapter.get_relation(identifier=target_relation.identifier ~ tmp_table_suffix,
                                              schema=schema,
                                              database=database) %}
-  {% set tmp_relation = make_temp_relation(target_relation, suffix=tmp_table_suffix) %}
+  {% set tmp_relation = athena__make_temp_relation(target_relation, suffix=tmp_table_suffix, temp_schema=tmp_schema) %}
```

Comment:
Above you overwrite `make_temp_relation`, I believe that here you can simply use make_temp_relation, because you are within the athena adapter it will be dispacted to `athena__make_temp_relation`. Could you try it out?